source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py

```diff
@@ -22,7 +22,17 @@
 import html
 from typing import Optional
 
-import iqs_client
+from iqs_client.models import ElementInCompartmentDescriptor,\
+    TypedElementInCompartmentDescriptor,\
+    QueryExecutionResponse,\
+    ModelCompartment,\
+    IndexMessage,\
+    SimpleMessage,\
+    QueryFQNList, \
+    GenericValidationResults,\
+    GenericValidationRule, \
+    ValidationDiagnostics
```

Comment:
such long import blocks sound very uncomfortable and brittle. Is there a specific reason that we cannot rely on `iqs_client.models`, perhaps with the alias `schema`?

Reply:
Note that the package `iqs_jupyter` already exposes this Python module as `schema`, and I believe most notebooks use something like `from iqs_jupyter import schema` and then `schema.ComputeCompartmentDeltaRequest` etc. We could similarly use the alias `schema` here.